### DISCUSSION TITLE
Don't Require SSO When Viewing Issues on Public Repository

### DISCUSSION THREAD

**Discussion by ificator** [community/community#13088] [Other Feature Feedback, Questions, & Ideas]
I work for Microsoft, which blocks authorization on non-secure devices such as my home PC. That means if I'm at home and want to simply [look at issues](https://github.com/OneDrive/onedrive-api-docs/issues) I can't, because SSO is forced and cannot be bypassed. Requiring sign-in for some level of interaction (assigning, tagging, closing, etc) makes sense, but I should not be blocked from performing operations that any ordinary Github user can do (viewing, commenting, etc).

**Comment by qiyuangong** [comment]
Same issue at Intel. I found that not all repos/orgs are impacted. 

Is there any configuration to turn off SSO on the public pages?

**Comment by Rigidity** [comment]
This has been extremely frustrating for the past few years, and now having a separate work computer makes it even worse.

**Reply by theficus** [reply]
I gave up and went back to having two accounts (a personal one and work one) to avoid this frustration. Of course it adds another level of frustration since now I have to deal with managing multiple accounts.
